thirdParty/alpaka/README.md

```diff
@@ -74,6 +74,8 @@ This library uses C++11 (or newer when available).
 |TBB 2.2+|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|:white_check_mark:|
 |CUDA 7.0+|:white_check_mark: (nvcc 7.0+)|:white_check_mark: (nvcc 8.0+)|:x:|:x:|:x:|:white_check_mark: (native/nvcc 8.0+)|:white_check_mark: (native)|:x:|:x:|
 
+*NOTE*: When natively compiling CUDA code with clang-3.8, the random functionality is not available.
```

Comment:
OPTIONAL:
better "the random number generation/functionality"

Reply:
This can only be changed in the alpaka repository. sry forgot to say it, there is no need to check the changes in this PR.